src/assets/js/multiple-authors.js

```diff
@@ -486,17 +486,27 @@ jQuery(document).ready(function ($) {
         });
     });
 
-    /**
-     * Add tab class to author editor's tr without tab
-     *
-     * This will add general tab class to 'Name' and Author URL
-     * or any tab that's rendered by default or third party
-     *  without tab attribute
-     */
     if ($('body').hasClass('taxonomy-author')) {
+        /**
+         * Add tab class to author editor's tr without tab
+         *
+         * This will add general tab class to 'Name' and Author URL
+         * or any tab that's rendered by default or third party
+         *  without tab attribute
+         */
         $('form#edittag tr.form-field:not(.ppma-tab-content)')
             .addClass('ppma-tab-content ppma-general-tab')
             .attr('data-tab', 'general');
+        
+        /**
+         * Add view link to author url field
+         */
+         $('form#edittag tr.form-field #slug')
+             .css({
+                 'width': '85%',
+                 'margin-right': '5px'
+             })
+             .after('<a href="' + MultipleAuthorsStrings.home_url + '/author/' + $("#slug").val() + '/" class="button-secondary" target="_blank">' + MultipleAuthorsStrings.view_text + '</a>')
```

Comment:
@ojopaul on a few sites, this line can be a problem. It creates the author URL in a way that will be invalid on sites that use plain permalinks.

The author URL should come from the object of `Author` class, `$author->link`. You can add it to the localize script array.

You can replicate it on a fresh WP site, or going to Settings > Permalinks > Plain. Save it and try to follow that View link.

Reply:
Sorry, my thinking wasn't full when i was creating this feature including the css and end up doing rubbish. I was creating it in such a way that when user modify the permalink, new permalink will be the one opened and when i realised it won't be possible as the page will lead to 404 without saving first, i change end result without considering other closed options are now opened

Reply:
@andergmartins, this PR is now updated